desktop/apps/artwork/client/index.coffee

```diff
@@ -161,5 +172,8 @@ module.exports =
       req: user: CurrentUser.orNull()
     }
       .then (data) ->
-        renderTemplates(data)
-        exec init
+        renderTemplates(extend data, {
+          auctionContextId: AUCTION.id
```

Comment:
Had to tweak this in https://github.com/artsy/force/pull/1045 , was causing a crash on non-auction artwork pages (`AUCTION` is undefined).